lib/web_ui/dev/web_engine_analysis.sh

```diff
@@ -17,4 +17,6 @@ echo "Running \`pub get\` in 'engine/src/flutter/lib/web_ui'"
 (cd "$WEB_UI_DIR"; $PUB_PATH get)
 
 echo "Running \`dart analyze\` in 'engine/src/flutter/lib/web_ui'"
-(cd "$WEB_UI_DIR"; $DART_PATH analyze --fatal-infos)
+(cd "$WEB_UI_DIR"; $DART_PATH analyze --fatal-infos lib/)
+(cd "$WEB_UI_DIR"; $DART_PATH analyze --fatal-infos test/)
+(cd "$WEB_UI_DIR"; $DART_PATH analyze --fatal-infos dev/)
```

Comment:
This change is so we don't analyze `build/` and `.dart_tool` directories (they don't exist on CI but they exist locally)

Reply:
_(Drive-by review, feel free to ignore)_

Why not `exclude` those two directories in the [analysis_options.yaml](https://github.com/flutter/engine/blob/master/lib/web_ui/analysis_options.yaml) file?

[Syntax reference](https://dart.dev/guides/language/analysis-options#excluding-files).

Reply:
Good find! Done.